UH-StudentServices/student_guide
# Feature/hub 410 chat

# Chat

- Display a chat widget on specified nodes

- Nodes are configured in `settings.local.php` (see `settings.local.php.j2` for the placeholder)

- The chat is displayed only when advisors are online (this is 100% controlled by the widget)

## Testing tips

- Configure some node IDs in `settings.local.php`

- Verify that the DOM contains a `div` having ID `chat-application`

- Verify that other nodes do not include the widget

- Verify that the correct language version of the chat is used (based on current language)
